code/splay.c

```diff
@@ -1237,7 +1237,7 @@ Bool SplayFindFirst(Tree *nodeReturn, SplayTree splay,
   while (!found) {
     Tree oldRoot, newRoot;
 
-    /* FIXME: Rename to "seen" and "not yet seen" or something. */
+    /* TODO: Rename to "seen" and "not yet seen" or something. */
```

Comment:
For what reason exactly?

Reply:
There's a general rule (possibly not documented) that we avoid the terms "old" and "new" because they often meaningless and relative. So this might been just that -- find better names specific to the context.  So:

m: TODO should say this.
mI: Add the rule about "old" and "new" probably to rule.impl.

Reply:
1. Fix: Clarified in 3c2c0797d0e9f8497b72ed0cf79ac2bf3c286c4e
2. Forget: Wasn't that relevant in this case. It'll come up again.